README.md

```diff
@@ -88,24 +88,46 @@ nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be i
 <a id="install-script"></a>
 ## Installing and Updating
 
-### Install & Update Script
+### Install NVM on macOS
+Change the ownership of these directories to your user
+```sh
+sudo chown -R $(whoami) /usr/local/share/zsh /usr/local/share/zsh/site-functions
+```
 
-To **install** or **update** nvm, you should run the [install script][2]. To do that, you may either download and run the script manually, or use the following cURL or Wget command:
+Make sure that user has write permission
 ```sh
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+chmod u+w /usr/local/share/zsh /usr/local/share/zsh/site-functions
 ```
+
+Use Humebrew to install NVM
```

Comment:
nobody should EVER use homebrew to install nvm; homebrew-installed nvm is 100% unsupported and discouraged.

Reply:
for me using macbook pro m1 it's the only way nvm work :(

Reply:
That shouldn’t be the case; node itself only supports m1 outside of Rosetta on v16+, but nvm itself should work fine.